src/attribution/onINP.ts

```diff
@@ -233,6 +235,100 @@ const getIntersectingLoAFs = (
   return intersectingLoAFs;
 };
 
+const getLoAFSummary = (attribution: INPAttribution) => {
+  const loafAttribution: LongAnimationFrameSummary = {};
+
+  // Stats across all LoAF entries and scripts.
+  const interactionTime = attribution.interactionTime;
+  const inputDelay = attribution.inputDelay;
+  const processingDuration = attribution.processingDuration;
+  let totalNonForcedStyleAndLayoutDuration = 0;
+  let totalForcedStyleAndLayout = 0;
+  let totalScriptTime = 0;
+  let numScripts = 0;
+  let slowestScriptDuration = 0;
+  let slowestScriptEntry!: PerformanceScriptTiming;
+  let slowestScriptPhase: string = '';
+  const phases = {} as Record<string, Record<ScriptInvokerType, number>>;
```

Comment:
```suggestion
  const phases: Record<string, Record<ScriptInvokerType, number>> = {};
```

but you may actually want `const phases: Record<string, Partial<Record<ScriptInvokerType, number>>> = {};`

Reply:
Changed to `Partial` without the `as`.